app/views/users/index.html.erb

```diff
@@ -1,72 +1,26 @@
-<section class="main mt-5">
+<section class="main mt-5"> 
+<h1>Hello</h1>
+   <% @users.each do |user| %>
   <div class="card mb-3">
     <div class="row g-0">
       <div class="col-md-4">
         <img src="https://www.fillmurray.com/640/360" class="img-fluid rounded-start" alt="...">
       </div>
       <div class="col-md-8">
         <div class="card-body">
-          <h5 class="card-title"><%= @user1.name%></h5>
-          <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-          <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="card mb-3">
-    <div class="row g-0">
-      <div class="col-md-4">
-        <img src="https://www.fillmurray.com/640/360" class="img-fluid rounded-start" alt="...">
-      </div>
-      <div class="col-md-8">
-        <div class="card-body">
-          <h5 class="card-title">Card title</h5>
-          <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-          <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="card mb-3">
-    <div class="row g-0">
-      <div class="col-md-4">
-        <img src="https://www.fillmurray.com/640/360" class="img-fluid rounded-start" alt="...">
-      </div>
-      <div class="col-md-8">
-        <div class="card-body">
-          <h5 class="card-title">Card title</h5>
-          <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-          <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-        </div>
-      </div>
+          <h5 class="card-title"> <%= userpost.title%></h5>
```

Comment:
Your views are generating errors. Kindly ensure that the variable `userpost` is correctly spelt as indicated in the attached screenshot below:


![image](https://user-images.githubusercontent.com/12745474/163673790-775a8e2a-547c-435f-9fa0-cf679bdb7dea.png)